superbench/runner/runner.py

```diff
@@ -121,7 +128,35 @@ def run(self):
                 continue
             benchmark_config = self._sb_benchmarks[benchmark_name]
             for mode in benchmark_config.modes or []:
-                if mode.name == 'torch.distributed':
+                if mode.name == 'local':
+                    logger.info('Runner is going to run %s.', benchmark_name)
+
+                    def run_proc(vars, rets):
+                        mode.update(vars)
+                        rc = self._ansible_client.run(
+                            self._ansible_client.get_shell_config(
+                                runner_command.format(
+                                    self.__get_mode_command(
+                                        mode, ('sb exec -c sb.config.yaml -C '
+                                               'superbench.enable={name}').format(name=benchmark_name)
+                                    )
+                                ),
+                                sudo=True
+                            )
+                        )
+                        rets[vars['proc_rank']] = rc
+
+                    jobs = []
+                    rets = multiprocessing.Manager().dict()
+                    for proc_rank in range(mode.proc_num or 1):
+                        proc = multiprocessing.Process(target=run_proc, args=({'proc_rank': proc_rank}, rets))
```

Comment:
Could we support both parallel and sequential execution? In some cases, parallel execution may have performance interference.

Reply:
added